.kitchen.yml

```diff
@@ -51,10 +51,12 @@ platforms:
       box: trombik/ansible-ubuntu-14.04-amd64
       box_check_update: false
 
-  - name: centos-7.2-x86_64
-    driver:
-      box: trombik/ansible-centos-7.2-x86_64
-      box_check_update: false
+# Fails with: "No package matching 'nsd' found available"
+# Maybe vagrant box issue. Disabling until it is fixed
+#  - name: centos-7.2-x86_64
+#    driver:
+#      box: trombik/ansible-centos-7.2-x86_64
+#      box_check_update: false
```

Comment:
because release 7.3 removed the package. `trombik/ansible-centos-7.2-x86_64` is actually 7.3 now. it might be confusing, but that is the way how they releases. see http://mirror.centos.org/centos/7.2.1511/readme